backend/api/Cargo.toml

```diff
@@ -25,10 +25,10 @@ tokio = { version = "1.25.0", features = ["full"] }
 serde = { version = "1.0.152", features = ["derive"] }
 hyper = { version = "0.14.24", features = ["full"] }
 common = { path = "../../common" }
-sqlx = { version = "0.6.2", features = ["postgres", "runtime-tokio-rustls", "json", "chrono"] }
+sqlx = { git="https://github.com/launchbadge/sqlx", branch="main", features = ["postgres", "runtime-tokio-native-tls", "json", "chrono"] }
```

Comment:
We will eventually move back to using a particular version when sqlx 0.7.1 gets released. 
We updated because sqlx 0.6.x does not support mTLS and also we switched from rustls to openssl because rustls does not support the ESDA cert (idk why)